code/addons/a11y/project.json

```diff
@@ -1,6 +1,8 @@
 {
-  "name": "@storybook/addon-a11y",
+  "name": "addon-a11y",
```

Comment:
@JReinhold @yannbf I tried moving the project.json to package.json, but somehow it is more limited (I can not override targets the same way as I can with project.json). Not sure if this is a limitation or a bug in NX, but I therefore reverted back to using project.json again.

One thing I found out is that we can remove the name of the project.json, which removes some redunancy. It will infer the name as being the directory name, which is the same convention that vitest uses, if you don't specify the project.